helm/alfresco-identity-service/values.yaml

```diff
@@ -27,6 +20,8 @@ realm:
     extraGroups:
       - name: testgroup
         path: "/testgroup"
+      - name: testgroup2
```

Comment:
Is this absolutely needed?

Reply:
Not needed, not sure why I added that actually